02_Methodiek.Rmd

```diff
@@ -0,0 +1,374 @@
+Er zou dan aan geen enkele cel een meetpunt kunnen toegewezen worden.
+
+Om dit euvel te ondervangen, werd een iteratie uitgevoerd. 
+Na de eerste ronde worden per GT-groep al de bekomen aantallen opgeteld en vergeleken met het gewenste aantal. 
+Is dat aantal te klein, dan wordt de benodigde oppervlakte een klein beetje verkleind (bijv. 9.9 ha per meetpunt) en wordt de berekening opnieuw uitgevoerd en het totaal aantal gecheckt.
+Dit gaat door tot wanneer het gewenst aantal bereikt wordt.
+De variatie in oppervlakten bepaalt de grootte van de verkleining. 
+Is de sprong voldoende klein, zal het bekomen aantal hooguit lichtjes hoger dan het gewenste aantal uitvallen.
+
+Figuur \@ref(fig:workflow-rastercellen) geeft een schematisch overzicht van de gevolgde werkgang.
+
+(ref:workflow-rastercellen) werkgang selectie rastercellen. Blauw gekleurde cirkels zijn brongegevens. De rode gekleurde zijn genereerde data. Het groen gekleurde is een (tussentijds) output-resultaat. De gele vakken stellen handelingen/bewerkingen voor.
+
+```{r workflow-rastercellen, fig.cap='(ref:workflow-rastercellen)'}
+include_graphics(path = file.path("figures", "workflow_rasterselectie.png"))
+```
+
+
+
+### Selectie van de meetpunten
+
+In dit deel bespreken we hoe we aan de gekozen rastercellen effectief meetpunten kunnen koppelen. 
+Een meetpunt is een locatie waar het (freatisch) grondwaterpeil periodiek gemeten wordt (of zal worden).
+In de praktijk gebeurt dat in een piëzometer of peilbuis.[^2]
+
+Elke locatie die tot één van de GT-groepen behoort, kan in principe in het meetnet opgenomen worden.
+Locaties waarvan het onduidelijk is tot welke GT-groep ze behoren worden niet meegenomen. 
+Dit kunnen locaties zijn die op grens liggen van habitatvlekken die tot een verschillende GT-groep behoren of het zijn locaties die in een habitatvlek gelegen zijn dat gekarteerd is als een complex van vegetaties die tot verschillende GT-groepen behoren.
+In de praktijk wordt deze voorwaarde toegepast door enkel die habitatvlekken en/of peilbuizen te selecteren die voor 50% of meer toegewezen kunnen worden aan eenzelfde GT-groep en die op minstens `r bufferpb` m afstand liggen van een andere GT-groep.
+
+[^2]: Normaal gesproken duidt in dit rapport de term meetpunt op de locatie eerder dan op het meetinstrument (peilbuis of piëzometer), maar soms slaat het op beide. Met peilbuis wordt in dit document ook een piëzometer bedoeld. 
+
+Figuur \@ref(fig:workflow-selectiepb-part1) geeft een schematisch overzicht van de werkgang die gevolgd werd om na te gaan welke peilbuizen beschikbaar zijn.
+
+(ref:workflow-selectiepb1) werkgang selectie rastercellen. Voor de betekenis van de symbolen zie figuur \@ref(fig:workflow-rastercellen).
+
+```{r workflow-selectiepb-part1, fig.cap='(ref:workflow-selectiepb1)'}
+include_graphics(path = file.path("figures", "workflow_pbselectie_part1.png"))
+```
```

Comment:
analoge opm als in vorige fig

Reply:
'één GT in een straal van 3 m': wellicht duidelijker indien je vooraan 'voorwaarde: slechts' toevoegt

Reply:
ok, figuur werd in die zin aangepast